apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx

```diff
@@ -69,9 +69,10 @@ export const SwipeUpDown = forwardRef<SwipeUpDownRef, SwipeUpDownProps>(
 
     const snapPoints = useMemo(() => {
       if (snapPointCount === 2) {
-        return [500, maxHeightOffset - tabBarHeight];
+        // TODO: use dynamic sizing on TransferOp screen
+        return [440, maxHeightOffset - tabBarHeight];
```

Comment:
If you make a payment link, then navigate to cancel it, is the cancel button still visible?

Reply:
yes, checked
![Simulator Screenshot - iPhone 14 Pro - 2024-04-03 at 14 41 17](https://github.com/daimo-eth/daimo/assets/6984346/eb828ef6-f776-4d53-aeac-4ffe2057ace1)